app/components/Receive/ReceivePanel/ReceivePanel.jsx

```diff
@@ -14,26 +18,48 @@ type Props = {
   address: string
 }
 
-export default class ReceivePanel extends React.Component<Props> {
-  canvas: ?Node
+type State = {
+  asset: ?string,
+  amount: ?number
+}
 
-  componentDidMount () {
-    QRCode.toCanvas(this.canvas, this.props.address, { version: 5 }, (err) => {
-      if (err) console.error(err)
-    })
+export default class ReceivePanel extends React.Component<Props, State> {
+  image: ?HTMLImageElement
+
+  state = {
+    asset: null,
+    amount: 0
   }
 
   render () {
     const { className, address } = this.props
+    const { asset, amount } = this.state
+    const symbols = ['-', 'NEO', 'GAS', ...Object.keys(TOKENS)]
```

Comment:
Could you use `ASSETS.NEO` and `ASSETS.GAS` for NEO/GAS

Reply:
updated